### DISCUSSION TITLE
part-ii/2017-119/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#381] [Question comments]
Part II 2017 Principles of Statistics: Paper 1, Section II, $28 \mathrm{~K}$ 

https://questions.tripos.org/part-ii/2017-119/

**Comment by ArthurConmy** [comment]
<details>
<summary>Spoiler</summary>

The idea is to use the example sheet result that general loss functions of the form $ L(\theta,a) = w(\theta) ( \theta - a )^2 $ have Bayes optimal estimators $ \mathbb{ E}^\Pi [ \theta w ( \theta ) ] / \mathbb{ E}^\Pi [ w(\theta) ] $. So Bayes optimal estimator is $x/n$ which is unbiased and hence the risk is $ \text{Var}( \hat{p}) / p(1-p) = \boxed{\frac{1}{n}}$. This is constant in $p$ and hence is minimax optimal.

</details>

**Reply by ArthurConmy** [reply]
Has anyone got a solution for the last part? It's not coming out for me.

**Reply by DannyDuberstein** [reply]
Not sure that is the right risk. Estimator is biased so MSE is variance, and dividing by p(1-p) makes it constant in p

**Reply by DannyDuberstein** [reply]
*unbiased

**Reply by DannyDuberstein** [reply]
I presume the x is mean to be a p in your expression, still not sure where it comes from though?
